lib/rules/utils/ast-utils.js

```diff
@@ -1200,6 +1200,23 @@ module.exports = {
         };
     },
 
+    /**
+     * Gets next location when the result is not out of bound, otherwise returns undefined.
+     * @param {SourceCode} sourceCode The sourceCode
+     * @param {{line: number, column: number}} location The location
+     * @returns {{line: number, column: number} | undefined} Next location
+     */
+    getNextLocation(sourceCode, location) {
+        const index = sourceCode.getIndexFromLoc(location);
+
+        // Avoid out of bound location
+        if (index + 1 > sourceCode.text.length) {
+            return void 0;
```

Comment:
According to our [code conventions](https://eslint.org/docs/developer-guide/code-conventions#primitive-literals) (which are difficult to find), I think we should return `null` here.

Reply:
I actually didn’t know this document existed...